### PR TITLE
[author][update] jquery-csv to 0.81

### PR DESCRIPTION
**Pull request by evanplaice**
Updated the source files and corrected the information in package.json.

The project migrated to GitHub, and the previous repo link pointed to an
 outdated unofficial fork.

### CODE REVIEW

File: ajax/libs/jquery-csv/package.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {
   "name": "jquery-csv",
-  "filename": "jquery.csv-0.71.min.js",
-  "version": "0.71",
-  "description": "CSV parser (complete, end-to-end, battle tested, performance optimized, 100% IETF RFC 4180 compatible) in pure js to be used with jQuery, nodejs and more",
-  "homepage": "http://code.google.com/p/jquery-csv/",
+  "filename": "jquery.csv-0.81.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-csv/package.json]: Should be `"filename": "jquery.csv.min.js",`

File: ajax/libs/jquery-csv/package.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+  "version": "0.81",
+  "description": "A jQuery CSV parser plugin. Battle Tested | Optimized | 100% IETF RFC 4180 Complete",
+  "homepage": "http://github.com/evanplaice/jquery-csv/",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-csv/package.json]: Hey let's use https here.

File: ajax/libs/jquery-csv/package.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
   "keywords": [
     "jquery",
     "csv"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/mirlord/jquery-csv"
+    "url": "http://github.com/evanplaice/jquery-csv/"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-csv/package.json]: Hey let's use https here.

**Comment by evanplaice** [comment]
@PeterDaveHello Done

**Comment by PeterDaveHello** [comment]
Seems you forgot you change the filename field, and also please squash/fixup these commits into only one commit, many thanks!

**Comment by evanplaice** [comment]
@PeterDaveHello I realized that after CI failed. I'll resubmit a squashed commit as requested.

**Comment by PeterDaveHello** [comment]
@evanplaice Thank you very much.

**Comment by PeterDaveHello** [comment]
@evanplaice if there is a problem, we can add it by our side, but it's up to you :smile:

**Comment by pvnr0082t** [comment]
@evanplaice Any update?

**Comment by PeterDaveHello** [comment]
@pvnr0082t I think you can take this.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Pull request closed without merging**